### ISSUE TITLE
Problem with pabot execution

### ISSUE THREAD

**Issue by bioneyez** [jz-jess/RobotEyes#52]
**Bug**
I have two problem using pabot with the RobotEyes library.
1. When I execute the tests using pabot with testlevelsplit then in the pabot_results folder the library generates an unnecessary folder with the test case name -1 character instead of using the existing one named same as the test case. 
For example in case of test case name **This is a dummy test**. The pabot_results will contain two folder: **This is a dummy test** and **This is a dummy tes**. The former will contain the _output.xml_, _robot_stderr.out_, _robot_stdout.out_ and every screenshot I do using the capture page screenshot keyword, and the latter will contain the visual_images folder.

2. After the execution finishes I should be able to generate the reports using the **reportgen** cmd but I would have to call this command for every test case since the visual_images folder is separated for every test case.

**To Reproduce**
Start the execution using pabot and testlevelsplit

**Expected behavior**
I want to be able to generate the report using the command only once (all the visual_images in the same folder separated by test cases, just like in case of **robot** cmd)

**Desktop:**
 - OS: Win 10
 - Browser Chrome
 - Version 74

**Comment by jz-jess** [comment]
@bioneyez The library isn't designed to work with pabot at a testlevel split. Any particular reason as to why you want use a test level split?

**Comment by bioneyez** [comment]
@jz-jess Yes. My test cases does not depend on each other so I want to execute them parallelly, but I don't want to make a test suite for each of them.

**Comment by jz-jess** [comment]
@bioneyez Implementing support for test level split might take time since I need to understand pabot's implementation of it.
Currently the library gets the test case name and output directory using robot framework's APIs.
Pabot executes each suite individually with a pybot/robot command and stores the results in a temp folder and merges them at the end. Library takes the path till `pabot_results` folder and then creates the `visual_images` folder at the parent of `pabot_results`. The images are then stored in the same fashion as in a sequential execution. I am not sure why the images are getting stored within pabot_results. Fixing this should ideally fix the duplicate test case folder creation issue as well.

I'd suggest that you use the default suite level split until its permanently fixed.

**Comment by bioneyez** [comment]
@jz-jess Alright. I'll try to find some time and help out if I can with this issue.

**Comment by jz-jess** [comment]
@bioneyez I finally took some time to investigate this. It looks like the issue was with pabot instead of the robot eyes.
I upgraded pabot to 1.8.1 which is the latest. Ran the following command:
`pabot --processes 2 --testlevelsplit -d results -v images_dir:base Tests`

Everything seems to have run fine for me. The `visual_images` folder got generated at the root of results folder and not within `pabot_results`. Inside `pabot_results` I could see folders 0, 1, 2 ... so on. 
Unless there is another way to reproduce this issue, I think testlevelsplit is compatible with RobotEyes.
